app/views/steps/evidence/upload/edit.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div>
+      <h1 class="govuk-heading-xl">
+        <%=t '.heading' %>
+      </h1>
+
+      <p><%= t('.list.intro') %></p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li><%= t('.list.option_1') %></li>
```

Comment:
I think is safe to assume this list will be dynamic. The design is a mock up. For now I propose hardcoding it in the view, we will figure out the logic to show "items" in the list later. Let's keep this focused and simple for now.

Reply:
Oh, I hadn't heard anything about it being dynamic. Makse sense - I'll change it back to hard code